Assets/Scripts/Actors/Player/PlayerInventory.cs

```diff
@@ -2,8 +2,10 @@
 using System.Collections;
 using System.Collections.Generic;
 
-[RequireComponent( typeof( PlayerCamera ), typeof( PlayerStats ) )]
-public class PlayerInventory : ActorComponent
+// types and current count
+[System.Serializable] public class InventoryDictionary : SerializableDictionary<InventoryItemData, int> { }
+
+    [RequireComponent( typeof( PlayerCamera ), typeof( PlayerStats ) )] public class PlayerInventory : ActorComponent
```

Comment:
Got some weird leading whitespace here. Also the `RequireComponent` attribute should be on the line before the class declaration.